sdk/Sdk.Generators/Sdk.Generators.csproj

```diff
@@ -9,16 +9,16 @@
     <RootNamespace>Microsoft.Azure.Functions.Worker.Sdk.Generators</RootNamespace>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
-    <MinorProductVersion>1</MinorProductVersion>
-    <PatchProductVersion>7</PatchProductVersion>
+    <MinorProductVersion>2</MinorProductVersion>
```

Comment:
This technically does not need a minor version rev (only patch) as it is a private asset being updated. But minor is fine as well.